config/credhub_secret.go

```diff
@@ -77,3 +88,11 @@ func (r *credHubSecret) Remove(configServerInstanceName string, credHubPath stri
 
 	return nil
 }
+
+func validateCredHubPath(credHubPath string) error {
+	matched, _ := regexp.MatchString(`\w+/\w+/\w+/\w+`, credHubPath)
```

Comment:
Is `\w` sufficient? It's only `[a-zA-Z0-9_]`... do we allow e.g. `-` too?

Reply:
Good point! Not sure about others, but definitely `-`.
What do you think about this one @spikymonkey?
```
[a-zA-Z0-9-_]+\/[a-zA-Z0-9-_]+\/[a-zA-Z0-9-_]+\/[a-zA-Z0-9-_]+
```